match/lib/match/storage.rb

```diff
@@ -39,7 +89,8 @@ def register_backend(type: nil, storage_class: nil, &configurator)
         end
       end
 
-      def for_mode(storage_mode, params)
+      def from_params(params)
```

Comment:
It seems like this change wasn't made backwards compatible 😬😬 
This was brought to my attention in https://github.com/fastlane/docs/pull/1225

Reply:
Is there anything I can do to help?

Reply:
Yeah fair enough. Not sure if we have any guidelines on what to consider important to keep backwards compatible and what not to, so that's to blame haha
Whenever I personally make changes or review code I try to keep _all_ APIs backwards compatible, but that may be exclusive to me and not really the team's preferences

I'll kick off a discussion around this in Slack so perhaps we can better define this :) 

Thanks for the input guys!